### PR TITLE
DB-921 - Unit Tests for New Features

### PR DESCRIPTION
**Pull request by drewfulton**
This covers Unit tests for the following:
- Paragraph Left/Right Margin & Indent
- Shape FlipH and FlipV
- BulletText, BulletColor, etc Classes through paragraph object
- Shape Styles

### CODE REVIEW

**Comment by drewfulton** [comment]
@luna215 I have split the larger Unit Tests tickets into 3 tickets so this one can be reviewed now and ideally added to the release tomorrow (Tuesday).  I've created two new tickets, one for all the Bullet component Classes and one for the table borders.  I'll do them as I have time but didn't want to hold this one up any longer.

File: Pipfile [hunk 1]
```diff
@@ -0,0 +1,19 @@
+[[source]]
```
Comment (luna215) [inline Pipfile:1]: Aren't these changes already in the master branch along with the circleci config file?

File: pptx/text/text.py [hunk 2]
```diff
@@ -582,7 +589,10 @@ def indent(self):
         margin of the paragraph.  If assigned |None|, the default value is
         used, 0.  Negative values can also be used.
         """
-        return self._pPr.indent
+        indent = self._pPr.indent
+        if indent is None:
+            return 0
```
Comment (luna215) [inline pptx/text/text.py]: I'm curious why you chose to return 0 instead of None? Is this how Steve is doing it?

**Review by luna215 — COMMENTED** [review]
The tests look good! 

I left a quick question and a comment about some changes that might already be in the master branch.

File: Pipfile (hunk 1 above)
Comment (drewfulton) [reply Pipfile:1]: nope, not for this repo, this didn't have a pipfile at all, just a `requirements.txt`.

File: pptx/text/text.py (hunk 2 above)
Comment (drewfulton) [reply pptx/text/text.py]: So effectively, yes that was how he was doing it, just not implemented like this.  After some more digging, I found there is a default value you can set for optional parameters in the `oxml` level of the code.  Implemented properly now and tests are passing.

**Comment by drewfulton** [comment]
Circle CI is now functioning!

**Review by luna215 — APPROVED** [review]
Looks good!

**Pull request merged**